### PR TITLE
retina icons for chrome omnibox

### PR DESCRIPTION
**Pull request by gmethvin**
@2is10

### CODE REVIEW

File: packrat/adapters/chrome/api.js [hunk 1]
```diff
@@ -365,7 +365,7 @@ api = function() {
       set: function(tab, path) {
         if (tab === pages[tab.id]) {
           tab.icon = path;
-          chrome.pageAction.setIcon({tabId: tab.id, path: path});
+          chrome.pageAction.setIcon({tabId: tab.id, path: {"19": path, "38": path}});
```
Comment (2is10) [inline packrat/adapters/chrome/api.js]: `{"38": path}`

File: packrat/adapters/chrome/manifest.json [hunk 2]
```diff
@@ -10,7 +10,10 @@
   },
   "options_page": "options.html",
   "page_action": {
-    "default_icon": "icons/keep.faint.png",
+    "default_icon": {
+      "19": "icons/keep.faint.png",
+      "38": "icons/keep.faint.png"
+    },
```
Comment (2is10) [inline packrat/adapters/chrome/manifest.json]: The `"19"` key-value pair should be omitted.

**Comment by 2is10** [comment]
Thanks.

File: packrat/adapters/chrome/manifest.json (hunk 2 above)
Comment (gmethvin) [inline packrat/adapters/chrome/manifest.json]: Okay. Didn't know if it'd work that way.

**Pull request merged**